docs/tutorials/microbench-tutorial.md

```diff
@@ -245,18 +248,18 @@ if __name__ == "__main__":
                     'results/X86/run_micro/{}/{}/{}'.format(bm,cpu,mem),
                     gem5_binary,gem5_repo,experiments_repo,
                     cpu,mem,os.path.join('microbench',bm,'bench.X86'))
-                run_gem5_instance.apply_async((run, os.getcwd()))
+                run.run()
```

Comment:
Do we want to show running a single experiment at a time (serially) like this, or the old way with celery?

It would be nice to decouple some of these things, but not required for this PR.